### ISSUE TITLE
TimePicker not working

### ISSUE THREAD

**Issue by aaj** [erikras/redux-form-material-ui#38]
I have a form with one TimePicker field, like so:

```
<form>
    <Field
        name='start_time'
        component={TimePicker}
    />
</form>
```

And I get this warning: 
Warning: Failed prop type: Required prop `component` was not specified in `Field`.

Followed by this error:
Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. Check the render method of `ConnectedField`.

If I change the component prop to DatePicker, it works fine.

**Comment by kjg531** [comment]
Seems that npm install for version 4.0.1 does not include TimePicker but it is in the GitHub repo.

**Comment by channikhabra** [comment]
@erikras Please publish to npm.

**Comment by gajus** [comment]
@erikras Is there any outstanding work that needs to be done before this can be published?

I am happy to contribute.

**Comment by timeswind** [comment]
how to fix this problem before next version release?

**Comment by erikras** [comment]
Fix published in [`v4.1.0`](https://github.com/erikras/redux-form-material-ui/releases/tag/v4.1.0).
